mkosi/resources/man/mkosi.md

```diff
@@ -562,6 +562,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     not specified and the directory `mkosi.output/` exists in the local
     directory, it is automatically used for this purpose.
 
+`OutputMode=`, `--output-mode=`, `-M`
+:   File system access mode used when creating the output image file or
+    directory. Takes an access mode in octal notation. If not set, uses the
```

Comment:
Using this for the directory seems wrong. When we're building a directory image this setting should not apply since it will directly affect the image itself as well